ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraModule.java

```diff
@@ -94,14 +93,6 @@ public void setMigrationFile(String migrationFile) {
 			this.migrationFile = migrationFile;
 		}
 
-		public Boolean getAutoMigrate() {
```

Comment:
You can drive it with adding or removing the binding of the migration module is what I was thinking, do you think it's worth keeping the ability to flag the migration off as well?